tests/setup-env.js

```diff
@@ -13,4 +13,5 @@ afterEach(() => {
   server.resetHandlers()
   DiscordManager.cleanup()
   jest.restoreAllMocks()
+  jest.useRealTimers()
```

Comment:
Let's also make sure we advance any existing timers (in case we forgot any):

```suggestion
  if (jest.isMockFunction(setTimeout)) {
    jest.runOnlyPendingTimers()
    jest.useRealTimers()
  } 
```